src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -0,0 +1,492 @@
+    protected static final SAMSequenceRecord CHR10_HG19 = new SAMSequenceRecord("chr10", 135534747);
+
+    // b36
+    protected static final SAMSequenceRecord CHR1_B36 = new SAMSequenceRecord("1", 247249719);
+    protected static final SAMSequenceRecord CHR2_B36 = new SAMSequenceRecord("2", 242951149);
+    protected static final SAMSequenceRecord CHR10_B36 = new SAMSequenceRecord("10", 135374737);
+
+    // b37
+    protected static final SAMSequenceRecord CHR1_B37 = new SAMSequenceRecord("1", 249250621);
+    protected static final SAMSequenceRecord CHR2_B37 = new SAMSequenceRecord("2", 243199373);
+    protected static final SAMSequenceRecord CHR10_B37 = new SAMSequenceRecord("10", 135534747);
+
+
+    public enum SequenceDictionaryCompatibility {
+        IDENTICAL,                      // the dictionaries are identical
+        COMMON_SUBSET,                  // there exists a common subset of equivalent contigs
+        NO_COMMON_CONTIGS,              // no overlap between dictionaries
+        UNEQUAL_COMMON_CONTIGS,         // common subset has contigs that have the same name but different lengths
+        NON_CANONICAL_HUMAN_ORDER,      // human reference detected but the order of the contigs is non-standard (lexicographic, for examine)
+        OUT_OF_ORDER,                   // the two dictionaries overlap but the overlapping contigs occur in different
+        // orders with respect to each other
+        DIFFERENT_INDICES               // the two dictionaries overlap and the overlapping contigs occur in the same
+        // order with respect to each other, but one or more of them have different
+        // indices in the two dictionaries. Eg., { chrM, chr1, chr2 } vs. { chr1, chr2 }
+    }
+
+    /**
+     * Tests for compatibility between two sequence dictionaries.  If the dictionaries are incompatible, then
+     * UserExceptions are thrown with detailed error messages.
+     *
+     * @param name1 name associated with dict1
+     * @param dict1 the sequence dictionary dict1
+     * @param name2 name associated with dict2
+     * @param dict2 the sequence dictionary dict2
+     * @param isReadsToReferenceComparison true if one of the dictionaries comes from a reads data source (eg., a BAM),
+     *                                     and the other from a reference data source
+     * @param intervals the user-specified genomic intervals: only required when isReadsToReferenceComparison is true,
+     *                  otherwise can be null
+     */
+    public static void validateDictionaries( final String name1,
+                                             final SAMSequenceDictionary dict1,
+                                             final String name2,
+                                             final SAMSequenceDictionary dict2,
+                                             final boolean isReadsToReferenceComparison,
+                                             final GenomeLocSortedSet intervals ) {
```

Comment:
Could you change the type of this last parameter from `GenomeLocSortedSet` to the more hellbender-friendly `List<SimpleInterval>`?